### DISCUSSION TITLE
downloading Pandoc and LaTex

### DISCUSSION THREAD

**Discussion by AztecWarlord** [Cyfrin/security-and-auditing-full-course-s23#195] [Q&A] [answered]
Is anyone having trouble downloading pandoc and LaTex for Ubuntu on WSL. Any assistance would be appreciated.

**Comment by EngrPips** [comment]
Hello @AztecWarlord, What issue did you run into while trying to download pandoc and Latex?

**Reply by AztecWarlord** [reply]
I am using WSL and am not sure if I use the linux or windows files?
The goal is to install from the command line

**Reply by AztecWarlord** [reply, marked as answer]
I installed pandoc without all the specs and it worked.

**Reply by EngrPips** [reply]
Awesome, I am glad you unblock yourself. Keep crushing it.
